kymatio/scattering2d/backend/tensorflow_backend.py

```diff
@@ -0,0 +1,176 @@
+# Authors: Edouard Oyallon, Sergey Zagoruyko
+
+import tensorflow as tf
+import numpy as np
+from collections import namedtuple
+
+BACKEND_NAME = 'tensorflow'
+
+def _iscomplex(x):
+    return x.dtype == np.complex64 or x.dtype == np.complex128
+
+def _isreal(x):
+    return x.dtype == np.float32 or x.dtype == np.float64
+
+class Pad(object):
+    def __init__(self, pad_size, input_size, pre_pad=False):
+        """
+            Padding which allows to simultaneously pad in a reflection fashion
+            and map to complex.
+            Parameters
+            ----------
+            pad_size : list of 4 integers
+                size of padding to apply.
+            input_size : list of 2 integers
+                size of the original signal
```

Comment:
Should we remove?

Reply:
@janden view?

Reply:
Why?

Reply:
@MuawizChaudhary ?

Reply:
Not used.

Reply:
right. Let's keep it, as it is used by other backends. Are you guys ok with this?

Reply:
Ok.